### ISSUE TITLE
During ACKNOWLEDGE get 'java.lang.IllegalStateException: Content stream is already disposed.'

### ISSUE THREAD

**Issue by egor-ryashin** [mesosphere/mesos-rxjava#73]
Time to time I got this exception trying to send ACKNOWLEDGE (not every time):
```java
Caused by: java.lang.IllegalStateException: Content stream is already disposed.
	at io.reactivex.netty.protocol.http.UnicastContentSubject$OnSubscribeAction.call(UnicastContentSubject.java:240)
	at io.reactivex.netty.protocol.http.UnicastContentSubject$OnSubscribeAction.call(UnicastContentSubject.java:213)
	at rx.Observable.unsafeSubscribe(Observable.java:9861)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.subscribe(Observable.java:9957)
	at rx.Observable.subscribe(Observable.java:9924)
	at rx.Observable.subscribe(Observable.java:9729)
	at rx.Observable.forEach(Observable.java:6781)
	at com.mesosphere.mesos.rx.java.SinkSubscriber.lambda$onNext$7(SinkSubscriber.java:66)
	at rx.internal.util.ActionSubscriber.onNext(ActionSubscriber.java:39)
	at rx.observers.SafeSubscriber.onNext(SafeSubscriber.java:134)
	at rx.internal.operators.OperatorObserveOn$ObserveOnSubscriber.call(OperatorObserveOn.java:227)
	... 9 more
```

**Comment by BenWhitehead** [comment]
Which version are you using so I can try and line up the line numbers accurately?

**Comment by egor-ryashin** [comment]
I'm working with
```maven
    <dependency>
      <groupId>com.mesosphere.mesos.rx.java</groupId>
      <artifactId>mesos-rxjava-protobuf-client</artifactId>
      <version>0.1.1</version>
    </dependency>
   
```

**Comment by BenWhitehead** [comment]
@egor-ryashin I'm going to publish a `1.2.0-SNAPSHOT` if you want to verify this fix before I release 1.2.0 (no scheduled date right now). Instructions are in the readme if you want to add the snapshot repo and test it out.

**Comment by BenWhitehead** [comment]
@egor-ryashin Version `0.1.2` has been released and is available in maven central.
